### PR TITLE
Scroll to view code panel as needed

### PR DESCRIPTION
**Pull request by burieberry**
The AI assistant chat panel will scroll when "View Code" button is pressed and if the full code editor is not already in view. (Addressed the issue mentioned here: https://github.com/cardstack/boxel/pull/1102#issuecomment-2015653801)

### CODE REVIEW

File: packages/host/app/components/matrix/room-message.gts [hunk 1]
```diff
@@ -108,7 +108,7 @@ export default class Room extends Component<Signature> {
             </Button>
             <div
               class='monaco-container'
-              {{setCssVar monaco-container-height=this.monacoContainerHeight}}
+              {{this.viewCodePanelModifier}}
```
Comment (backspace) [inline packages/host/app/components/matrix/room-message.gts]: Does it make sense to describe what the modifier does vs where it’s used?

**Review by backspace — APPROVED** [review]
I tried this locally, works well 🎉

File: packages/host/app/components/matrix/room-message.gts (hunk 1 above)
Comment (burieberry) [reply packages/host/app/components/matrix/room-message.gts]: Any suggestions? `panelHeightAndScrollModifier`? I'm terrible with naming

Comment (backspace) [reply packages/host/app/components/matrix/room-message.gts]: `scrollBottomIntoView`? 🤔

**Review by lukemelia — APPROVED** [review]

**Pull request merged**